step-sequencer/index.html

```diff
@@ -304,7 +304,7 @@ <h2>DTMF</h2>
       durControl.addEventListener(
         "input",
         (ev) => {
-          noiseDuration = parseInt(ev.target.value, 10);
+          noiseDuration = ev.target.value;
```

Comment:
`ev.target.value` is of type `string`:

```suggestion
          noiseDuration = parseFloat(ev.target.value);
```